static/js/place/chart.tsx

```diff
@@ -451,7 +424,40 @@ class Chart extends React.Component<ChartPropType, ChartStateType> {
     }
     this.embedModalElement.current.show(
       svgXml,
-      this.dataCsv(),
+      () => {
```

Comment:
That is a good callout. I'm inclined to just leave it with the new CSV export behavior for now, and add a TODO to address changing the denominator in a follow-up PR. I'd argue that even without the per-capita calculation, the new CSV export is still a step in the right direction since it gives more transparency as to where the data came from.